game/src/devtools/fifteen_min/mod.rs

```diff
@@ -68,14 +73,43 @@ impl State<App> for Viewer {
 
         if ctx.redo_mouseover() {
             self.hovering_on_bldg = match app.mouseover_unzoomed_buildings(ctx) {
-                Some(ID::Building(id)) => Some(HoverOnBuilding {
-                    id,
-                    tooltip: if let Some(time) = self.isochrone.time_to_reach_building.get(&id) {
-                        Text::from(Line(format!("{} away", time)))
-                    } else {
-                        Text::from(Line("This is more than 15 minutes away"))
-                    },
-                }),
+                Some(ID::Building(hover_id)) => {
+                    let map = &app.primary.map;
+                    let path_request = PathRequest {
+                        start: map.get_b(self.start_building_id).sidewalk_pos,
+                        end: map.get_b(hover_id).sidewalk_pos,
+                        constraints: PathConstraints::Pedestrian,
+                    };
+
+                    let drawn_route = map
+                        .pathfind(path_request)
+                        .and_then(|path| path.trace(map, Distance::ZERO, None))
+                        .map(|polyline| {
+                            let dashed_lines = polyline.dashed_lines(
```

Comment:
Maybe we could just display dashed while zoomed in, and thicker `polyline.make_polygons(...)` unzoomed?